Domaci_ukol_Automatizovane_testovani_Cvetelina/src/main/java/com/cvetelina/book/tools/Chapter1Page.java

```diff
@@ -13,199 +13,148 @@
 
 public class Chapter1Page {
     private static WebDriver driver;
+    private static final String SUB_HEADING_CHAPTER1 = "If you have arrived here then you have installed Selenium IDE and are ready to start recording your first test.";
+    private static final String ANOTHER_PAGE_TEXT = "The following text has been loaded from another page on this site. It has been loaded in an asynchronous fashion so that we can work through the AJAX section of this chapter";
+    private static final String LOAD_TEXT_WITH_BUTTON = "To be used after the AJAX section of the book\n" + "I have been added with a timeout\n";
+    private static final By DROPDOWN_SELECTOR = By.id("selecttype");
+    private static final By EDITABLE_CONTENT_FIELD = By.id("html5div");
 
     @BeforeClass
     public void beforeClassSetup() {
         driver = initializeDriver();
         goToHomePage(driver);
 
-        System.out.println("Redirects to the chapter1 page after click.");
         WebElement chapter1Link = driver.findElement(By.linkText("Chapter1"));
         chapter1Link.click();
+        System.out.println("Redirects to the chapter1 page after click.");
+        String URL = driver.getCurrentUrl();
+        Assert.assertEquals(URL, "http://book.theautomatedtester.co.uk/chapter1");
+        driver.manage().timeouts().pageLoadTimeout(10, TimeUnit.SECONDS);
     }
 
     @Test
-    public void checkSubHeadingAppearsTest() {
+    public void checkSubHeadinDisplayedTest() {
         WebElement subHeading = driver.findElement(By.xpath("//*[@class='mainbody']//p[1]"));
-        System.out.println("Expected text is:");
-        System.out.println(subHeading.getText());
-        Assert.assertEquals("If you have arrived here then you have installed Selenium IDE and are ready to start recording your first test.", subHeading.getText());
+        System.out.println("Actual text is:" + subHeading.getText());
+        Assert.assertEquals(SUB_HEADING_CHAPTER1, subHeading.getText());
     }
 
     @Test
     public void checkRadioButtonTest() {
         WebElement radioButton = driver.findElement(By.id("radiobutton"));
-        System.out.println("Button is displayed.");
         Assert.assertTrue(radioButton.isDisplayed());
-        for (int i = 0; i < 2; i++) {
-            radioButton.click();
-
-            Assert.assertTrue(radioButton.isSelected());
-            System.out.println("Button is toggled on.");
-            Assert.assertTrue(!radioButton.isSelected(), "Button is not toggled off, as expected.");
-            System.out.println("Button is toggled off");
+        radioButton.click();
+        Assert.assertTrue(radioButton.isSelected());
```

Comment:
Add message which logs what happened when radioButton.isSelected() equals FALSE.

Reply:
Assert.assertTrue(radioButton.isSelected(), "Radio button is not selected!")